lib/config/urc.js

```diff
@@ -37,10 +38,8 @@ module.exports = class Urc {
   normalize() {
     const urc = this;
     // Normalize the merged configuration
-    if (urc.siteBasePath && urc.siteBasePath !== '/') {
-      urc.siteBasePath = urc.siteBasePath.replace(/\/$/, '');
-      urc.siteBasePath = path.join('/', urc.siteBasePath);
-    }
+    urc.siteBasePath = urlJoin('/', urc.siteBasePath);
```

Comment:
Are you sure that we'll end up with normalized `/` at the *end* of the string?

Reply:
I guess we don't need `urc.siteBasePath` to end with `/`. If at some place (Webpack's publicPath) we can append it there.